src/character.cpp

```diff
@@ -256,16 +256,17 @@ void Character::recalc_sight_limits()
         else {
         sight_boost = sight_boost_cap;
         }
-    } else if (has_trait("ELFA_NV")) {
+    } else if (has_active_mutation("ELFA_NV")) {
         sight_boost = 6; // Elf-a and Bird eyes shouldn't coexist
-    } else if (has_trait("NIGHTVISION2") || has_trait("FEL_NV") || has_trait("URSINE_EYE")) {
+    } else if (has_active_mutation("NIGHTVISION2") || has_active_mutation("FEL_NV") ||
+        has_trait("URSINE_EYE")) {
```

Comment:
OK, here's the other half of the thing.  It's entirely possible to test for mutations being Active or not, so the Ursine one needed a bit more finesse.  It can stay active, but only *test for active* ( `has_active_mutation` ) here in the NV code, not in the sight-range code.  Sight range would remain simply checking for the mutation being present, `has_trait` .